p4proto/kctrl/README.md

```diff
@@ -1,7 +1,10 @@
 <!--
 /*
+ * Copyright (c) 2013-2021 Barefoot Networks, Inc.
```

Comment:
For Readme, this is not required. Please remove

Reply:
Fixed